README.md

```diff
@@ -92,6 +93,29 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion (experimental)
+
+The ability to convert existing SBOMs means you can create SBOMs in different formats quickly, without the need to regenerate the SBOM from scratch, which may take significantly more time.
+
+```
+syft convert <ORIGINAL-SBOM-FILE> -o <NEW-SBOM-FORMAT>[=<NEW-SBOM-FILE>]
+```
+
+This feature is experimental and data might be lost when converting formats. Packages are the main SBOM component easily transferable across formats, whereas files and relationships, as well as other information Syft doesn't support, are more likely to be lost. 
+
+We support formats with wide community usage AND good encode/decode support by Syft. The supported formats are:
+- Syft JSON
+- SPDX 2.2 JSON
+- SPDX 2.2 tag-value
+- CycloneDX 1.4 JSON
+- CycloneDX 1.4 XML
+
+Conversion example:
+```sh
+syft alpine:latest -o json=sbom.syft.json # generate a syft SBOM
```

Comment:
Should -o here be `json=sbom.syft.json`?

I thought valid values for this were `syft-json cyclonedx-xml cyclonedx-json github github-json spdx-tag-value spdx-json table text`

Reply:
`json` is an alias for `syft-json`

Reply:
just like you can use `-o json` in syft...

Reply:
So in this case the output file is `sbom.syft.json`? I understand `json` as the alias. What was confusing was that the command is outputting to a file called `sbom.syft.json`

Reply:
No need to hole up the PR, but =<file> might be misleading without comments on where the output is going. Especially since we have other demos/examples where syft is just doing a `syft packages -o json alpine:latest > syft-sbom.json`

Reply:
FYI this is documented in the Syft readme: https://github.com/anchore/syft#multiple-outputs

Reply:
Having said that, I might also just use a "more normal" `-o <format>` either with `> <file>` or `--file <file>`

Reply:
This is a very minor issue to me I wouldn't hold up the PR for, though.